client/signup/controller.js

```diff
@@ -99,7 +102,9 @@ export const removeP2SignupClassName = function () {
 export default {
 	redirectTests( context, next ) {
 		const currentFlowName = getFlowName( context.params );
-		currentFlowName === 'onboarding' && loadExperimentAssignment( 'refined_reskin_v1' );
+		currentFlowName === 'onboarding' &&
+			loadExperimentAssignment( 'refined_reskin_v1' ) &&
+			loadExperimentAssignment( 'design_picker_after_onboarding' );
```

Comment:
```suggestion
		currentFlowName === 'onboarding' && loadExperimentAssignment( 'refined_reskin_v1' );
		(currentFlowName === 'onboarding' || currentFlowName === 'with-design-picker') && loadExperimentAssignment( 'design_picker_after_onboarding' );
```

We want to use the same eligibility for prefetching as for running the experiment.

Reply:
Actually this prefetching isn't doing much as we are blocking basically right here, this can probably be removed.